selfdrive/car/gm/interface.py

```diff
@@ -247,6 +247,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.steerActuatorDelay = 0.2
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
+    elif candidate == CAR.ESCALADE_ESV_2019:
+      ret.minEnableSpeed = -1.  # engage speed is decided by pcm
+      ret.mass = 2739.
+      ret.wheelbase = 3.302
+      ret.steerRatio = 17.3
+      ret.centerToFront = ret.wheelbase * 0.5
+      ret.steerActuatorDelay = 0.2
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.tire_stiffness_factor = 1.0
```

Comment:
```suggestion
      ret.tireStiffnessFactor = 1.0
```

Reply:
oops. so many refactors happened since I first drafted this branch 😅